### ISSUE TITLE
PlayerSensor improvement

### ISSUE THREAD

**Issue by mburger82** [austinv11/PeripheralsPlusPlus#169]
Suggestion:

### Description
The PlayerSensor is a really handy tool for command-computers aswell. 
So it would be ideal, if you also could give back the player coordinates.
And really important would be the dimension, the player is in.

**Expected behavior:** 
new Method: 
```
getPlayerData(playerName)
{
    x
    y
    z
    dimension
}
```

**Comment by dmhzmxn** [comment]
I agree, if you could pull x y z coordinates and get if the player is crouching or is on the ground and an is player like in openccsensors player sensor. it is sometimes unreliable and just removing openccsensors would be great.
however, keep the x y z relative to the sensor unless passed otherwise.
and lastly the ability to filter names of entity's or their entity id
function getPlayerData(playerName, posIsRelative, nameOrEntityID)
